css/properties/text-decoration-skip.json

```diff
@@ -116,7 +106,40 @@
               "opera": "mirror",
               "opera_android": "mirror",
               "safari": {
-                "version_added": "≤13.1"
+                "version_added": "7"
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror",
+              "webview_ios": "mirror"
+            },
+            "status": {
+              "experimental": true,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "objects": {
+          "__compat": {
```

Comment:
This value appears to not be mentioned in the spec? Accordingly, shouldn't we set `standard_track: false`?